projects/mercury/src/metadata/views/ViewFacets.js

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import withStyles from '@mui/styles/withStyles';
+import {Button, Grid} from '@mui/material';
+import {alpha} from '@mui/material/styles';
+import {ofBooleanValueType, ofRangeValueType} from "./metadataViewUtils";
+import Facet from './MetadataViewFacetFactory';
+import type {MetadataViewFacet, MetadataViewFilter, MetadataViewOptions, ValueType} from "./MetadataViewAPI";
+
+const styles = theme => ({
+    confirmFiltersButtonBlock: {
+        bottom: 0,
+        marginTop: 8,
+        marginLeft: 4,
+        width: 253
+    },
+    confirmFiltersButtonBlockActive: {
+        position: 'sticky',
+        backgroundColor: alpha(theme.palette.common.white, 0.8)
+    },
+    confirmFiltersButton: {
+        width: '100%'
+    },
+    facet: {
+        borderColor: theme.palette.primary.light,
+        borderWidth: 1.5,
+        borderRadius: 6
+    },
+    facetHeaders: {
+        textAlign: 'left',
+        marginTop: 6,
+        fontSize: 13,
+        color: theme.palette.primary.light,
+        marginLeft: 4
+    }
+});
+
+export const ViewFacets = (props) => {
```

Comment:
> to keep the naming consistent
I did it on purpose, because these two are sub-components which are now nicely grouped, but I agree with the consistency argument, will change it